src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java

```diff
@@ -111,7 +112,20 @@ public static void verifyPluginNodeIsDisplayedOnCanvas(String pluginName, long t
    * @param pluginName
    */
   public static void navigateToPluginPropertiesPage(String pluginName) {
-    ElementHelper.hoverOverElement(CdfStudioLocators.locatePluginNodeInCanvas(pluginName));
+    int attempts = 0;
+    while (attempts < 5) {
```

Comment:
please add a comment explaining why this is necessary (basically what you have added in the PR description)

Reply:
nit: can we use a constant for max attempts?

Reply:
1. Comment added explaining need. 
2. Used constant for max attempts.